cli/src/ui.rs

```diff
@@ -397,6 +421,10 @@ impl Ui {
         }
     }
 
+    pub fn movement_edit_mode(&self) -> MovementEditMode {
+        self.movement.edit.to_owned()
+    }
```

Comment:
Despite the config name, it's unrelated to the `Ui` object. Move it to movement_util.rs?

Reply:
Done.

OT: Btw, is there a bug filed to make config loading single homed? `UserSettings` seems like the correct location. If there is already an issue filed for this? If nobody owns it, I'd like to take it after this issue is closed, else, I'll file one and assign to myself.

Reply:
I don't remember if there was the exact issue, but iirc we'd rather want separate config definitions/structs (per feature or something.) A single big config definition doesn't work because extensions might need their own settings.

Reply:
I see. This makes sense. Especially since commands have access to config via `command.settings().config()`, so it's not really a big deal.

Thanks for the context.